apps/mark-scan/frontend/src/api.ts

```diff
@@ -183,9 +181,11 @@ export const endCardlessVoterSession = {
     const queryClient = useQueryClient();
     return useMutation(apiClient.endCardlessVoterSession, {
       async onSuccess() {
-        // Because we poll auth status with high frequency, this invalidation isn't strictly
+        // Because we poll auth status with high frequency, auth invalidation isn't strictly
         // necessary
-        await queryClient.invalidateQueries(getAuthStatus.queryKey());
+        await queryClient.invalidateQueries({
+          queryKey: [getAuthStatus.queryKey(), getInterpretation.queryKey()],
```

Comment:
I think you need two separate calls. The `queryKey` array is a single query key filter (see https://tanstack.com/query/v4/docs/react/guides/query-invalidation#query-matching-with-invalidatequeries)

Reply:
[d2285b0](https://github.com/votingworks/vxsuite/pull/3855/commits/d2285b03eda5a239cbbad0a87d0746e6d127d037)